src/main/java/core/basesyntax/figures/Rectangle.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.figures;
+
+public class Rectangle extends Figure {
+    private int width;
+    private int height;
+
+    public Rectangle(int width, int height, String color) {
+        this.width = width;
+        this.height = height;
+        this.setColor(color);
```

Comment:
Why do you set color via setter, please pay attetion that color is the field of Figure class